test/specs/mounting-options/scopedSlots.js

```diff
@@ -0,0 +1,40 @@
+import { describeWithShallowAndMount, vueVersion } from '~resources/utils'
+import ComponentWithScopedSlots from '~resources/components/component-with-scoped-slots.vue'
+
+describeWithShallowAndMount('scopedSlots', (mountingMethod) => {
+  if (vueVersion >= 2.5) {
+    it('mounts component scoped slots', () => {
+      const wrapper = mountingMethod(ComponentWithScopedSlots, {
+        scopedSlots: {
+          'item': '<p slot="item" slot-scope="props">{{props.index}},{{props.text}}</p>'
```

Comment:
Do we need to include slot=item in the markup? We already identify it as the item slot in the scopedSlots object